packages/stateful/actions/core/dao_governance/BecomeSubDao/index.tsx

```diff
@@ -0,0 +1,76 @@
+import { FamilyEmoji } from '@dao-dao/stateless'
+import {
+  ActionComponent,
+  ActionContextType,
+  ActionKey,
+  ActionMaker,
+  UseDecodedCosmosMsg,
+  UseDefaults,
+} from '@dao-dao/types/actions'
+import { makeWasmMessage } from '@dao-dao/utils'
+
+import { AddressInput } from '../../../../components'
+import { BecomeSubDaoComponent, BecomeSubDaoData } from './Component'
+
+const defaultBecomeSubDaoData = {
+  admin: '',
+}
+
+const useDefaults: UseDefaults<BecomeSubDaoData> = () => defaultBecomeSubDaoData
+
+const useDecodedCosmosMsg: UseDecodedCosmosMsg<BecomeSubDaoData> = (
+  msg: Record<string, any>
+) => {
+  try {
+    return {
+      match: true,
+      data: {
+        admin: msg.wasm.execute.nominate_admin.admin,
+      },
+    }
+  } catch (e) {
```

Comment:
```suggestion
  } catch {
```

drop the unused variable